core/Db/Schema/Mariadb.php

```diff
@@ -9,11 +9,18 @@
 
 namespace Piwik\Db\Schema;
 
+use Piwik\Date;
+
 /**
  * Mariadb schema
  */
 class Mariadb extends Mysql
 {
+    public function getDatabaseType(): string
+    {
+        return 'MariaDb';
```

Comment:
It seems officially it's
```suggestion
        return 'MariaDB';
```